locksmith/src/controllers/priceController.ts

```diff
@@ -15,6 +17,40 @@ namespace PriceController {
     const pricing = await keyPricer.generate(lockAddress)
     return res.json(pricing)
   }
+
+  // This method will return the key price in USD by default, but can eventually be used to return prices in a different curreny (via query string)
+  export const fiatPrice = async (
+    req: Request,
+    res: Response
+  ): Promise<any> => {
+    const { lockAddress } = req.params
+    const keyPricer = new KeyPricer(
+      config.web3ProviderHost,
+      config.unlockContractAddress
+    )
+
+    // First, let's see if the lock was authorized for credit card payments
+    const isAuthorizedForCreditCard = await AuthorizedLockOperations.hasAuthorization(
+      lockAddress
+    )
+
+    if (!isAuthorizedForCreditCard) {
```

Comment:
I think it might be worthwhile to throw an error for this case.  It should make it bit easier for the caller to react rather than doing a null check

Reply:
So, eventually I want us to be able to have multiple fiat currencies returned, so maybe instead of `null` we just yield an empty object?